.commitlintrc.yml

```diff
@@ -0,0 +1,7 @@
+# Rules for commitlint
+# Reference of all rules: https://commitlint.js.org/reference/rules.html
+# This file adds customizations rules to the default ones defined in https://www.npmjs.com/package/@commitlint/config-conventional
+
+extends: ["@commitlint/config-conventional"]
+rules:
+  "subject-case": [2, "always", ["sentence-case"]]
```

Comment:
Is it possible to pass these parameters in the command line?
Aren't these parameters used only in the workflow? Will they be used by a dev? In my opinion, i don't think so.